lasagne/__init__.py

```diff
@@ -2,6 +2,17 @@
 Tools to train neural nets in Theano
 """
 
+try:
+    import theano
+except ImportError:  # pragma: no cover
+    raise ImportError("""Could not import Theano.
+
+Please make sure you install a recent enough version of Theano.  See
+section 'Install from PyPI' in the installation docs for more details.
+""")
```

Comment:
Just for convenience, I'd replace the last `.` by a colon and add this URL: http://lasagne.readthedocs.org/en/latest/user/installation.html#install-from-pypi
This error message is targeted at users just doing `pip install lasagne`, and those may not know about the docs at all yet, so we can help them a little.